assets/component-swatch-input.css

```diff
@@ -10,6 +10,12 @@
   forced-color-adjust: none;
 }
 
+@media screen and (max-width: 750px) {
```

Comment:
You don't need the @media query here, we already override the size for the larger screens here: https://github.com/Shopify/dawn/blob/436e1045db73d1dec121dc4f1151984fab516c86/assets/component-product-variant-picker.css#L153-L156